src/amber/cli/templates/app/.gitignore.ecr

```diff
@@ -5,7 +5,7 @@
 /.vscode/
 /tmp/
 .env
-.amber_secret_key
+.encryption_key
```

Comment:
`.encryption_key` is more descriptive than `.amber_secret_key`.

This is a change I've been wanting to make for a while but haven't because it was a breaking change. Now seemed like a good time since environment loader change and format of env.yml's is already a breaking change.